packages/frontend-app-api/src/wiring/parameters.ts

```diff
@@ -251,8 +289,33 @@ export function mergeExtensionParameters(options: {
     })),
   ];
 
+  // Install all extension overrides
+  for (const extension of overrideExtensions) {
+    // Check if our override is overriding an extension that already exists
+    const index = configuredExtensions.findIndex(
+      e => e.extension.id === extension.id,
+    );
+    if (index !== -1) {
+      // Only implementation, attachment point and default disabled status are overridden, the source is kept
+      configuredExtensions[index].extension = extension;
+      configuredExtensions[index].params.attachTo = extension.attachTo;
+      configuredExtensions[index].params.disabled = extension.disabled;
+    } else {
+      // Add the extension as a new one when not overriding an existing one
+      configuredExtensions.push({
+        extension,
+        params: {
+          source: undefined,
```

Comment:
Asking just to make sure I understand the logic 🙂 :
We don't have access to the source because plugins can't provide overrides, right?

Reply:
Yep we could say that the `ExtensionOverride` object is the source, but that doesn't have any valuable information right now so I figure'd it's better to just leave the source out for now

Reply:
Got it, thanks!